docs/src/main/sphinx/release/release-440.md

```diff
@@ -0,0 +1,70 @@
+# Release 440 (7 Mar 2024)
+
+## General
+
+* Add support for sub-queries inside `UNNEST` clauses. ({issue}`17953`)
+* Improve performance of {func}`arrays_overlap`. ({issue}`20900`)
+* Change the default value for the `jmxExport` resource group property to
+  `true`. ({issue}`20810`)
+* Fix query failure when a `CHECK` constraint is null. ({issue}`20906`)
+* Fix query failure for `CASE` aggregation queries when the input evaluation
```

Comment:
There's no such a thing as a "CASE aggregation query". Replace it with something like "aggregations over conditional expressions" or "aggregations over CASE expressions".